packages/react-devtools-extensions/README.md

```diff
@@ -7,15 +7,28 @@ The easiest way to install this extension is as a browser add-on:
 * [Firefox Add-ons](https://addons.mozilla.org/en-US/firefox/addon/react-devtools/)
 
 ## Development
-
 You can also build and install this extension from source.
 
-DevTools embeds local versions of several NPM packages also in this workspace. If you have not already built them, you'll need to do that before getting started by running the following command in the root directory of this repository:
+### Prerequisite steps
+DevTools depends on local versions of several NPM packages<sup>1</sup> also in this workspace. You'll need to either build or download those packages first.
+
+<sup>1</sup> Note that at this time, an _experimental_ build is required because DevTools depends on the `createRoot` API.
+
+#### Build from source
+To build dependencies from source, run the following command from the root of the repository:
 ```sh
-yarn build -- react,react-dom,react-is,scheduler --type=NODE
+RELEASE_CHANNEL=experimental \
```

Comment:
I keep making versions of this line for myself too, seems like a useful enough command to have its own alias in scripts? (Maybe not in this PR, and I’m happy to do so myself)

Reply:
Sure. I'll add a new alias to package for DevTools. Not sure that's exactly what you're suggesting, but it may be worth doing anyway.

Reply:
Lemme think about it some more? Just seems so common, but also tribal/hearsay knowledge.

Reply:
Okay. For now though I added a `yarn build-for-react` command.